language-guides/java-third-party/java-third-party.adoc

```diff
@@ -0,0 +1,143 @@
+
+[cols="1,4"]
+|===
+| icon:user[] Authors | Developers from http://www.larus-ba.it/neo4j/en/[Larus BA Italy^] and Neo4j
+| icon:gift[] Package | https://github.com/neo4j-contrib/neo4j-jdbc/releases/latest
+| icon:github[] Source | https://github.com/neo4j-contrib/neo4j-jdbc
+| icon:book[] Docs | https://github.com/neo4j-contrib/neo4j-jdbc/blob/master/README.adoc
+| icon:play-circle[] Example | {examples}/movies-java-jdbc
+| icon:book[] Blog Post | http://neo4j.com/blog/couchbase-jdbc-integrations-neo4j-3-0/
+|===
+
+include::../_includes/community-driver-notice.adoc[]
+
+
+[#java-jcypher]
+=== JCypher
+
+JCypher provides seamlessly integrated Java access to Neo4j at different levels of abstraction:
+
+At the topmost level of abstraction, JCypher allows you to *map complex business domains* to graph databases.
+You can take an arbitrarily complex graph of domain objects or POJOs and store it in a straightforward way into Neo4j for later retrieval. 
+You do not need to modify your domain object classes in any way, there is also no need for annotations. 
+JCypher provides a default mapping out of the box.
+
+At the same level of abstraction, *Domain Queries* provide the power and expressiveness of queries on a graph database, 
+while being formulated on domain objects or on types of domain objects, respectively.
+The true power of Domain Queries comes from the fact, that the graph of domain objects is backed by a graph database.
+
+At the next lower level of abstraction – access to graph databases – is provided based on a generic graph model.
+While simple, the model allows you to easily navigate and manipulate graphs. 
+
+At the bottom level of abstraction, a *native Java DSL* in the form of a fluent Java API allows you to intuitively and comfortably formulate queries against Neo4j.
+
+:maven-jcypher: http://search.maven.org/#search|gav|1|g%3A%22net.iot-solutions.graphdb%22%20AND%20a%3A%22jcypher%22
+
+[cols="1,4"]
+|===
+| icon:user[] Author | https://github.com/Wolfgang-Schuetzelhofer[Wolfgang Schützelhofer^]
+| icon:gift[] Package | link:{maven-jcypher}[http://maven.org^]
+| icon:github[] Source | http://github.com/Wolfgang-Schuetzelhofer/jcypher
+| icon:book[] Docs | http://jcypher.iot-solutions.net/
+| icon:book[] Article | https://neo4j.com/blog/jcypher-focus-on-your-domain-model-not-how-to-map-it-to-the-database/
+|===
+
+[#neo4j-grails]
+=== Groovy & Grails: Neo4j Grails Plugin
+
+image::{img}grails.png[width=200,float=right]
+
+The goal of GORM for Neo4j is to provide a 'as-complete-as-possible' GORM implementation that maps domain classes and instances to the Neo4j nodespace. The following features are supported:
+
+* Marshalling from Neo4j Nodes to Groovy types and back again
+* Support for GORM dynamic finders, criteria and named queries
+* Session-managed transactions
+* Access to Neo4j's traversal capabilities
+* Access the Neo4j graph database in all flavours (Embedded, REST and HA)
```

Comment:
We should avoid using the term HA, as that was our legacy clustering technology. If it works with clusters, then let's just go with that wording.

Reply:
Thanks for the ping: It is now Bolt only.